src/tests/createHeadlessForm.test.js

```diff
@@ -776,6 +780,74 @@ describe('createHeadlessForm', () => {
       });
     });
 
+    it('support "radio" field boolean type', () => {
+      const { fields, handleValidation } = createHeadlessForm(schemaInputTypeRadioBoolean);
+
+      expect(fields).toMatchObject([
+        {
+          description: 'Are you over 18 years old?',
+          label: 'Over 18',
+          name: 'over_18',
+          options: [
+            {
+              label: 'Yes',
+              value: true,
+            },
+            {
+              label: 'No',
+              value: false,
+            },
+          ],
+          required: true,
+          schema: expect.any(Object),
+          type: 'radio',
+        },
+      ]);
+
+      assertOptionsAllowed({
+        handleValidation,
+        fieldName: 'over_18',
+        validOptions: [true, false],
+        type: 'boolean',
```

Comment:
suggestion: Would you agree this is better?
```suggestion
        type: schemaInputTypeRadioBoolean.properties.over_18.type,
```